src/mumble/AudioOutputSample.h

```diff
@@ -55,6 +55,7 @@ class AudioOutputSample : public AudioOutputUser {
 
 	bool bLoop;
 	bool bEof;
+	bool m_stopped;
```

Comment:
```suggestion
	std::atomic<bool> m_stopped;
```
On pretty much all platforms, this should essentially make no difference as operations on bool variables typically already are atomic. But technically, we're accessing this variable from different threads and thus being explicit here seems to be better.